templates/bundles/PrestaShopBundle/Admin/Sell/Catalog/Product/Category/category_tree_form.html.twig

```diff
@@ -0,0 +1,29 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+{% form_theme categoryTreeSelectorForm '@PrestaShop/Admin/Sell/Catalog/Product/Form/categories_form_theme.html.twig' %}
```

Comment:
okay, that will make it easier to understand

Reply:
ok I tried a little, but there is one thing - the tags component is reused in both forms, so its either we need 3 form themes (so that 2 of them can inherit from the tags one) or we just keep it in one file. I think it might be easier when everything is in one place. Though tell me if you think Im wrong